addons/test_discuss_full/tests/test_performance.py

```diff
@@ -678,6 +678,7 @@ def test_init_messaging(self):
                     'uuid': channel_livechat_2.uuid,
                 },
             ],
+            'companyName': 'YourCompany',
```

Comment:
not working on runbot, I guess just write the env.company.name value at the top of the test to a determined value